### PR TITLE
Redirect resource service

### PR DESCRIPTION
**Pull request by paulchiu**

### CODE REVIEW

File: src/AdminApi/Redirect/Builders/GetRedirectsRequest.php [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+ * @see https://help.shopify.com/api/reference/redirect#index
+ */
+class GetRedirectsRequest implements PagingRequestBuilderInterface
+{
+    use AuthorizedRequestBuilderTrait;
+
+    const URI_TEMPLATE = 'https://%s.myshopify.com/admin/redirects.json';
+
+    /** @var array|int[] $ids */
+    protected $ids;
```
Comment (Jore) [inline src/AdminApi/Redirect/Builders/GetRedirectsRequest.php]: The `id`'s parameter isn't documented on Shopify and testing came back negative.

File: src/AdminApi/Redirect/Builders/GetRedirectsRequest.php [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    }
+
+    /**
+     * @return array
+     */
+    public function toArray(): array
+    {
+        $array = [];
+
+        if (!empty($this->ids)) {
```
Comment (Jore) [inline src/AdminApi/Redirect/Builders/GetRedirectsRequest.php]: Same as above comment.

**Comment by Jore** [comment]
Just a single issue I see.

File: src/AdminApi/Redirect/Builders/GetRedirectsRequest.php (hunk 1 above)
Comment (paulchiu) [reply src/AdminApi/Redirect/Builders/GetRedirectsRequest.php]: Fixed in commit 5ebffee

File: src/AdminApi/Redirect/Builders/GetRedirectsRequest.php (hunk 2 above)
Comment (paulchiu) [reply src/AdminApi/Redirect/Builders/GetRedirectsRequest.php]: Fixed in commit 5ebffee

**Comment by Jore** [comment]
LGTM. Thanks.

**Pull request merged**